### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by Ethan-Bit-256** [premake/premake-core#1697] [Q&A] [answered]
How do i get myself added to the contributors list? I'm very good with C, fairly decent with Lua, and have used premake for a handful of projects so I'm good with it!

**Comment by nickclark2016** [comment, marked as answer]
Having a PR merged to mainline is how to get listed as a contributor.

**Reply by Ethan-Bit-256** [reply]
Thanks!!!
